rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -776,15 +776,38 @@ public static byte[] serializeCoinbaseInformation(CoinbaseInformation coinbaseIn
     }
 
     public static byte[] serializeSha256Hash(Sha256Hash hash) {
+        if (hash == null){
+            return new byte[]{};
+        }
         return RLP.encodeElement(hash.getBytes());
     }
 
-    public static Sha256Hash deserializeSha256Hash(byte[] data) {
+    public static Optional<Sha256Hash> deserializeSha256Hash(byte[] data) {
         RLPElement element = RLP.decodeFirstElement(data, 0);
         if (element == null) {
-            return null;
+            return Optional.empty();
+        }
+        return Optional.of(Sha256Hash.wrap(element.getRLPData()));
+    }
+
+    public static Optional<Keccak256> deserializeKeccak256(byte[] data) {
+        if (data == null) {
+            return Optional.empty();
+        }
+
+        RLPList rlpList = (RLPList) RLP.decode2(data).get(0);
+        if (rlpList.size() != 1) {
+            throw new RuntimeException(String.format("Invalid serialized keccak256. Expected 1 element, but got %d", rlpList.size()));
+        }
+
+        return Optional.of(new Keccak256(rlpList.get(0).getRLPRawData()));
```

Comment:
Just to get an empty array when array has length equal to 0, instead of getting null. 

Implementation:
![image](https://user-images.githubusercontent.com/94651730/185167531-c9a01a5e-c1ff-434f-a55c-9d7144f4fdc0.png)

Reply:
I see, thanks!